packages/hackathon/src/scenes/JudgingEditorScene/components/JudgingViewDialog.tsx

```diff
@@ -23,4 +23,31 @@
  SOFTWARE.
 
  */
-export * from './ProjectViewScene'
+import type { FC } from 'react'
+import React from 'react'
+import { Dialog, DialogHeader, DialogContent } from '@looker/components'
+import type { IJudgingProps } from '../../../models'
+import { JudgingView } from './JudgingView'
+
+interface JudgingViewDialogProps {
+  judging?: IJudgingProps
+  closer: () => void
+}
+
+export const JudgingViewDialog: FC<JudgingViewDialogProps> = ({
+  judging,
```

Comment:
`JudgingViewDialog` props aren't clear about their behavior. The existence of judging determines if the dialog is open or closed. Is the props to follow the old props of `ProjectViewDialog`? 

Minor, but i'd recommend naming `closer` to `onClose` since it's an event listener and passing it through.

Both are minor comments, not blocking.